### PR TITLE
Publish packages to npmjs

### PR DESCRIPTION
**Pull request by cristidas**
- Publish `sdk/sdk-common` and 'sdk/sdk-client' as packages to npmjs
- Trigger automatically on version changes to `main` branch or manually for code on `dev` and `main` branches`

### CODE REVIEW

**Review by nickkounz — APPROVED** [review]

File: sdk/sdk-client/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@summerfi/sdk-client",
```
Comment (piotrwitek) [inline sdk/sdk-client/package.json:2]: revert please, this will not be publised

File: sdk/sdk-common/package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@summerfi/sdk-common",
```
Comment (piotrwitek) [inline sdk/sdk-common/package.json:2]: revert please

File: .github/workflows/publish-client-package.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        node-version: '20'
+        registry-url: 'https://registry.npmjs.org'
+        corepack: true
+
+    - name: Install pnpm
+      run: corepack enable && corepack prepare pnpm@latest --activate
+
+    - name: Install dependencies with pnpm
+      run: pnpm i
+      working-directory: sdk/sdk-client/bundle
```
Comment (piotrwitek) [inline .github/workflows/publish-client-package.yaml:30]: pnpm i at the monorepo root level and also pnpm build

File: .github/workflows/publish-client-package.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+    - name: Publish package with pnpm
+      run: pnpm bundle:npm
+      working-directory: sdk/sdk-client/bundle
```
Comment (piotrwitek) [inline .github/workflows/publish-client-package.yaml:34]: change to sdk/sdk-client

File: .github/workflows/publish-client-package.yaml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+    - name: Install pnpm
+      run: corepack enable && corepack prepare pnpm@latest --activate
+
+    - name: Install dependencies with pnpm
+      run: pnpm i
+      working-directory: sdk/sdk-client/bundle
+
+    - name: Publish package with pnpm
+      run: pnpm bundle:npm
```
Comment (piotrwitek) [inline .github/workflows/publish-client-package.yaml:33]: change to pnpm publish:npm

File: .github/workflows/publish-common-package.yaml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+    - name: Install dependencies with pnpm
+      run: pnpm i
+      working-directory: sdk/sdk-common/bundle
+
+    - name: Publish package with pnpm
+      run: pnpm bundle:npm
+      working-directory: sdk/sdk-common/bundle
+      env:
+        NPM_ACCESS_TOKEN: ${{ inputs.npm_access_token }}
```
Comment (piotrwitek) [inline .github/workflows/publish-common-package.yaml:36]: same changes as in publish-client-package.yaml

**Review by piotrwitek — CHANGES_REQUESTED** [review]

**Review by piotrwitek — APPROVED** [review]
looks good thanks

**Pull request merged**